packages/@ourworldindata/core-table/src/CoreColumnDef.ts

```diff
@@ -76,16 +75,12 @@ export interface CoreColumnDef extends ColumnColorScale {
     color?: Color // A column can have a fixed color for use in charts where the columns are series
 
     // Source information used for display only
-    sourceName?: string
-    sourceLink?: string
-    dataPublishedBy?: string
-    dataPublisherSource?: string
-    retrievedDate?: string
-    additionalInfo?: string
-    attribution?: string
+    source?: OwidSource
     timespanFromMetadata?: string
 
+    // Metadata v2
     origins?: OwidOrigin[]
+    presentation?: Omit<OwidVariablePresentation, "topicTagsLinks">
```

Comment:
Why is this typed to omit the topicTagsLinks? It will have them at runtime, no?

Reply:
yeah it does have access to topicTagsLinks at run time. it's just not supposed to use it... should I simplify?

Reply:
Why is it not supposed to use it?

Reply:
Don't know, I guess it can. I just didn't think topic page related info is useful to Grapher

Reply:
I think with the sources/about tab in particular we see that this is not a clear line to cut. I appreciate your instinct towards compartmentalizing but I think there is a good chance that in places like the about tab in particular we'll want to make use of all the metadata we have. E.g. tags could be used to group indicators or provide inline links to topic pages. I think I would just pass the full object through.

Reply:
Makes sense!